tools/clang/test/HLSL/pix/DebugVSParameters.hlsl

```diff
@@ -0,0 +1,17 @@
+// RUN: %dxc -Emain -Tvs_6_0 %s | %opt -S -hlsl-dxil-debug-instrumentation,parameter0=1,parameter1=2 | %FileCheck %s
+
+// Check that the instance and vertex id are parsed and present:
+
+// CHECK: %PIX_DebugUAV_Handle = call %dx.types.Handle @dx.op.createHandle(i32 57, i8 1, i32 0, i32 0, i1 false)
+// CHECK: %VertId = call i32 @dx.op.loadInput.i32(i32 4, i32 0, i32 0, i8 0, i32 undef)
+// CHECK: %InstanceId = call i32 @dx.op.loadInput.i32(i32 4, i32 1, i32 0, i8 0, i32 undef)
+// CHECK: %CompareToVertId = icmp eq i32 %VertId, 1
```

Comment:
Are you getting any trouble in release builds with names here?

Reply:
Release builds worked fine. These are names I asserted myself, so they don't get re-used and appended-with-numbers. Not that I got release correct first time: I had to modify some of the checks to remove auto-generated numerical names for variables and labels.